app/Jobs/SendEmptyWikibaseNotificationsJob.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Notifications\EmptyWikibaseNotification;
+use App\Wiki;
+use App\WikibaseNotificationSentRecord;
+use Carbon\CarbonImmutable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Arr;
+
+class SendEmptyWikibaseNotificationsJob extends Job implements ShouldBeUnique
+{
+    public function handle (): void
+    {
+        $wikis = Wiki::with(['notification', 'wikiLifecycleEvents'])->get();
```

Comment:
I don't think preloading `notification` is needed as the lookup happens by querying `wikibase_notification_sent_records` directly?